### DISCUSSION TITLE
Changing aria-label value per event on multi-event listing

### DISCUSSION THREAD

**Discussion by curtismchale** [add2cal/add-to-calendar-button#573] [Q&A] [answered]
On pages with more than one event, and thus more than one `atcb` button I don't see a way to set a custom `aria-label` per event so that screen reader users can know which event they're adding to their calendar. Right now every button shows up for screen readers with the same text of `Add to Calendar`

![same-aria-label](https://github.com/add2cal/add-to-calendar-button/assets/74561/8d935cad-6993-47d8-bddd-92f8373d19a8)

**Comment by jekuer** [comment, marked as answer]
Optimized with v2.6.9 - however, the whole ARIA labeling will require additional polishing in the future.

**Reply by curtismchale** [reply]
Thanks for the quick turn around. Helps us with accessibility work.
